components/ldap.rst

```diff
@@ -0,0 +1,66 @@
+.. index::
+   single: Ldap
+   single: Components; Ldap
+
+The Ldap Component
+==================
+
+    The Ldap component provides a means to connect to an Ldap server (OpenLdap or Active Directory).
```

Comment:
The component is indeed standalone. The security component actually is the one which integrates LDAP functionality in Symfony.